R/utils.R

```diff
@@ -235,6 +242,43 @@ find_csv_file <- function(filename, optional, quiet = FALSE) {
   }
 }
 
+#' Normalize file path to the working directory
```

Comment:
Since you've added rmarkdown anyway, see `markdown::relative_to()`–could this be used instead?

Reply:
Currently rmarkdown is just a suggestion. This is a good idea - do you think it's worth adding another dependency?

Reply:
For now, `drake` is just a suggestion too, so it seems OK to me. It's easy to imagine that we'll want to use `rmarkdown` to generate other reports as well. Thoughts @pralitp ?

Reply:
I think the difference is that because it is being called from `find_csv_file` adding the `rmarkdown`  may actually break some users set up who just want to use plain `driver()`.  And sometimes updating packages isn't something to to take lightly.  That is my only concern.

Reply:
Fair point. Maybe log this in an issue to come back to later, but leave alone for now.